docs/standard/serialization/system-text-json-converters-how-to.md

```diff
@@ -347,6 +347,16 @@ The converter can deserialize JSON that was created by using the same converter
 
 The converter code in the preceding example reads and writes each property manually. An alternative is to call `Deserialize` or `Serialize` to do some of the work. For an example, see [this StackOverflow post](https://stackoverflow.com/a/59744873/12509023).
 
+Another alternative way to call `Deserialize` in the `Read` method is to:
```

Comment:
nit: this isn't an alternative way to call `Deserialize` (since the first example didn't re-enter the serializer) - it's a alternative approach that involves using the serializer.